src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserManagementResource.kt

```diff
@@ -179,11 +179,7 @@ class UserManagementResource(
     @Schema(description = "Username", example = "testuser1", required = true)
     @PathVariable @Size(max = 30, min = 1, message = "username must be between 1 and 30") username: String,
     @Schema(description = "Password", example = "HeLl0W0R1D", required = true)
-    @Pattern(
-      regexp = "^[A-Za-z0-9]{14,30}$",
```

Comment:
oracle validates the password anyway so this just added an extra layer of complexity - especially when minimum should have been 9 for general users